src/classes/HiFiCommunicator.ts

```diff
@@ -257,6 +257,54 @@ export class HiFiCommunicator {
         });
     }
 
+    /**
+     * Adjusts the perceived volume of another user for this communicator's current connection only.
+     * This can be used to provide a more comfortable listening experience for the client. If you need to perform moderation actions which apply to all users, use the {@link https://docs.highfidelity.com/rest/latest/index.html|Administrative REST API}.
+     * 
+     * To use this command, the communicator must currently be connected to a space. You can connect to a space using {@link connectToHiFiAudioAPIServer}.
+     * 
+     * @param hashedVisitId  The hashed visit ID of the user whose volume will be adjusted.
+     * Use {@link addUserDataSubscription} and {@link HiFiCommunicator.onUsersDisconnected} to keep track of the hashed visit IDs of currently connected users.
+     *
+     * @param gain  The relative volume to apply to the other user. By default, this is `1.0`. The gain can be any value greater or equal to `0.0`.
+     * For example: a gain of `2.0` will double the volume of the user, while a gain of `0.5` will halve the user's volume. A gain of `0.0` will effectively mute the user.
+     * 
+     * @returns If this operation is successful, the Promise will resolve with `{ success: true, personalVolumeAdjustResponse: <The response to `audionet.personal_volume_adjust` from the server in Object format> }`.
+     * If unsuccessful, the Promise will reject with `{ success: false, error: <an error message> }`.
+     */
+    async adjustPersonalVolume(visitIdHash: String, gain: Number): Promise<any> {
```

Comment:
I'm a little concerned that this method name will imply that you're adjusting your OWN volume, rather than someone else's volume. (Also we should probably explicitly say 'gain' rather than 'volume'.) Thoughts for alternatives? Maybe just `adjustOtherUserGain`?

Reply:
I agree that using the term `gain` instead would be better for the sake of more consistent terminology.

I also see the potential for confusion with the use of the word, "personal." I would like to convey somehow that the gain change of the other user only applies to this particular client's stream only. Something like, "adjust another user's gain, but just for me." I'll need to think about it a little bit.

Reply:
Thoughts on these? They are a bit verbose:

- `setOtherUserGainForMyStream`
- `setOtherUserGainLocally`

Reply:
`setOtherUserGainForThisConnection` maybe? Naming things is HARD.